docs-2.0/nebula-console.md

```diff
@@ -0,0 +1,178 @@
+# Nebula Console
+
+Nebula Console is a native CLI client for Nebula Graph, it can be used to connect a Nebula Graph cluster and execute queries, while supporting special commands to manage parameters, export CSV files, export DOT files, import test datasets, etc. This topic describes these special commands.
+
+To connect Nebula Graph using Nebula Console, see [Step 3: Connect to Nebula Graph](2.quick-start/3.connect-to-nebula-graph.md).
+
+!!! note
+
+    The commands are case insensitive.
+
+## Manage parameters
+
+You can save parameters for parameterized queries.
+
+!!! note
+
+    - Parameters are not supported in VID.
+
+    - Parameters are not supported in sample clause.
+
+    - Saving multiple parameters at a time is not supported.
+
+    - Parameters are not retained after the session is released.
+
+The command to save a parameter is as follows:
+
+```ngql
+nebula> :param <param_name> => <param_value>;
+```
+
+The example is as follows:
+
+```ngql
+nebula> :param p1 => "Tim Duncan";
+nebula> MATCH (v:player{name:$p1})-[:follow]->(n)  RETURN v,n;
++----------------------------------------------------+-------------------------------------------------------+
+| v                                                  | n                                                     |
++----------------------------------------------------+-------------------------------------------------------+
+| ("player100" :player{age: 42, name: "Tim Duncan"}) | ("player125" :player{age: 41, name: "Manu Ginobili"}) |
+| ("player100" :player{age: 42, name: "Tim Duncan"}) | ("player101" :player{age: 36, name: "Tony Parker"})   |
++----------------------------------------------------+-------------------------------------------------------+
+nebula> :param p2 => {"a":3,"b":false,"c":"Tim Duncan"};
+nebula> RETURN $p2.b AS b;
++-------+
+| b     |
++-------+
+| false |
++-------+
+```
+
+The command to view the saved parameters is as follows:
+
+```ngql
+nebula> :params;
+```
+
+## Export a CSV file
+
+CSV files save the returned result of a executed command.
+
+!!! note
+
+    - A CSV file will be saved in the working directory, i.e., what the linux command `pwd` shows.
+
+    - This command only works for the next query statement.
+
+The command to export a csv file is as follows:
+
+```ngql
+nebula> :CSV <file_name.csv>
```

Comment:
```suggestion
nebula> :CSV <file_name.csv>;
```

Reply:
add a semicolon at the end of a statement. please take a look at the full content